From ba269f41de23bdcd062584e543707b6596474f85 Mon Sep 17 00:00:00 2001
From: Gabriel Briones Sayeg <gabriel.briones.sayeg@intel.com>
Date: Wed, 14 Mar 2018 15:44:41 -0600
Subject: [PATCH] Remove SASL dependency

Signed-off-by: Gabriel Briones Sayeg <gabriel.briones.sayeg@intel.com>
---
 setup.cfg | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.cfg b/setup.cfg
index 4b2427a..f45599c 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -1,5 +1,5 @@
 [_ldap]
-defines = HAVE_SASL HAVE_TLS HAVE_LIBLDAP_R
+defines = HAVE_TLS HAVE_LIBLDAP_R
 extra_compile_args = 
 extra_objects = 
 libs = ldap_r lber
-- 
2.16.2

